src/tir/transforms/hoist_if_then_else.cc

```diff
@@ -277,18 +327,21 @@ class HoistCandidateSelector final : public StmtExprVisitor {
     return false;
   }
 
-  std::vector<const ForNode*> ordered_for_list_;
+  std::vector<const Object*> ordered_list_;
```

Comment:
Can you add a comment here to clarify that there can only be `ForNode` and `AttrNode` in `ordered_list_`?